docs/user/monitoring/kibana-alerts.asciidoc

```diff
@@ -0,0 +1,37 @@
+[role="xpack"]
+[[kibana-alerts]]
+= {kib} Alerts
+
+The {stack} {monitor-features} provide
+<<alerting-getting-started,{kib} alerts>> out-of-the box to notify you of
+potential issues in the {stack}. These alerts are preconfigured based on the
+best practices recommended by Elastic. However, you can tailor them to meet your 
+specific needs.
+
+When you open the *{stack-monitor-app}* app, the preconfigured {kib} alerts are
+created automatically. If you collect monitoring data from multiple clusters,
+these alerts can search, detect, and alert on various conditions across the
+clusters. The alerts are visible alongside your existing {watcher} cluster
+alerts. You can view details about the alerts that are active and view health
+and performance data for {es}, {ls}, and Beats in real time, as well as
+analyze past performance. You can also modify active alerts in the
```

Comment:
Suggest removing "in the Stack Monitoring app" from this sentence.

Reply:
Done, thanks!